app/views/users/index.html.erb

```diff
@@ -1,41 +1,24 @@
 <h1>Users</h1>
-<h3>Admins</h3>
+<h3 id="admins">Admins</h3>
+<p><a href="#reps">Housing Committee</a> / <a href="#students">Students</a></p>
```

Comment:
can we add the jumping nav to the top of the page and have a "Back to top" link under each header instead?